examples/operator/convolution_operator.py

```diff
@@ -0,0 +1,55 @@
+"""Create a convolution operator by wrapping a library."""
+
+import odl
+import scipy
+import scipy.signal
+
+
+class Convolution(odl.Operator):
+    """Operator calculating the convolution of a kernel with a function.
+
+    The operator inherits from ``odl.Operator`` to be able to be used with ODL.
+    """
+
+    def __init__(self, kernel):
+        """Initialize a convolution operator with a known kernel."""
+
+        # Store the kernel
+        self.kernel = kernel
```

Comment:
I'd advocate to at least use a guaranteed type (`np.ndarray`) for storage, storing as-is is bad practice.

Reply:
I'll leave this open now to not spam this too much. The implementation currently requires that we can do `kernel.space` which limits the use-cases and the user couldn't feed an ndarray anyway.

I'm trying to keep this somewhat short, more of an example of how someone would use ODL and less a perfect example of proper coding style

Reply:
OK